recipes/cmaes/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "cmaes" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: c5529b94d6739569b635cc33649c924ce038f3519f6a2db003ce368602d40cbe
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - numpy
+    - python
+
+test:
+  imports:
+    - cmaes
+  source_fileese:
```

Comment:
Since you are only doing an import test and not running the test suite, you don't need the `source_files` section and `requires`

Reply:
Thank you for your swift review! I forgot to specify a command. I'll push the following change after passed local tests.

```diff
diff --git a/recipes/cmaes/meta.yaml b/recipes/cmaes/meta.yaml
index c212fb9d4..a9a1bee8e 100644
--- a/recipes/cmaes/meta.yaml
+++ b/recipes/cmaes/meta.yaml
@@ -16,7 +16,6 @@ build:

 requirements:
   host:
-    - numpy
     - pip
     - python
   run:
@@ -26,7 +25,9 @@ requirements:
 test:
   imports:
     - cmaes
-  source_fileese:
+  commands:
+    - "{{ PYTHON }} setup.py test"
+  source_files:
     - tests
   reqiures:
     - optuna
```